LICENSE

```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2016 Rodrigo Moraes. All rights reserved.
```

Comment:
Yes, "The Gorilla Authors" is the right credit. I wonder how I missed the license file.

muxy is "done" in terms of API and basic implementation, but there are fine details to take care (decoding encoded path variables and options like strict slashes). Most importantly, there are currently 0 lines of tests and docs. You are welcome!